internal/otg_helpers/otg_flow_helpers/otgflowhelpers.go

```diff
@@ -0,0 +1,216 @@
+// Package otgflowhelpers provides helper functions to create different flows on traffic generator.
+package otgflowhelpers
+
+import (
+	"github.com/open-traffic-generator/snappi/gosnappi"
+)
+
+// Iana Ethertype is the IANA Ethertype for MPLS, IPv4 and IPv6.
+const (
+	IanaMPLSEthertype = 34887
+	IanaIPv4Ethertype = 2048
+	IanaIPv6Ethertype = 34525
+)
+
+/*
+Flow is a struct to hold Flow parameters.
+TxNames and RxNames should be set to a valid OTG endpoint name.
+Creting basic IPv4 Flow.
+
+top = gosnappi.NewConfig()
+
+	FlowIPv4 = &Flow{
+	  TxNames:   []string{"interface1"},
+	  RxNames:   []string{"interface2"},
+	  FrameSize: 1500,
+	  FlowName:  "IPv4Flow",
+	  EthFlow:   &EthFlowParams{SrcMAC: "00:11:22:33:44:55", DstMAC: "00:11:22:33:44:66"},
+	  IPv4Flow:  &IPv4FlowParams{IPv4Src: "12.1.1.1", IPv4Dst: "11.1.1.1", IPv4DstCount: 10},
```

Comment:
Please change the example to use IP's from the contributing guide

https://github.com/openconfig/featureprofiles/blob/9dbadd5440efaa355392e4c9536b8b07c12d440e/CONTRIBUTING.md?plain=1#L338-L372